cli.ts

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
```

Comment:
Is this a valid directive in a `.ts` file? Personally, I'm not familiar with this which is why I'm asking.

Reply:
I did see this when I glanced at this yesterday with fresh eyes. It was from your original implementation, but upon further inspection we don't really need it anymore. It's called a [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) and it essentially tells the compiler which interpreter and args to use. Since we're not running this with `./cli.ts` (we use `bun ./cli.ts`) we don't really need it anymore. I'll remove this

Reply:
Roger, thanks Duncan! Haha it has been so long I've seen this code I forgot that I added that shebang. Thanks for linking that article, that's an interesting read.